force-app/main/default/classes/BotTest.cls

```diff
@@ -1,6 +1,16 @@
 @isTest
 public class BotTest {
     
+    static testMethod void testAutoBotController() {
+        //My new test method
+		Bot_Command__c bc1 = new Bot_Command__c(Sample_Utterance__c='help lightning', apex_class__c='HandlerHelpTopic', pattern__c='help (.*)');
```

Comment:
Variable `Bot_Command__c(Sample_Utterance__c='help lightn...` is created directly in a test method. Please use a data builder or factory to centralise creation of test data.